### DISCUSSION TITLE
Is it compatible with Govee?

### DISCUSSION THREAD

**Discussion by lordofscripts** [e154/smart-home#278] [Q&A]
I have a bunch of GOVEE devices, which are serviceable via their API. Is the `Smart Home` software compatible with those devices (**brand**: _Govee_)?

**Comment by e154** [comment]
At the moment, this software package cannot manage govee devices without intermediaries.

If there is no support from any manufacturer through a plugin, then it is possible to control via the rest API or the command line in scripts.

How do I see a solution to this situation?
1. You can control external solutions via API or calls via command line from scripts.
2. I can buy one device and write a plugin to work with the govee devices in two weeks, if you help me with debugging
